### DISCUSSION TITLE
Compare Home Asstant Panels

### DISCUSSION THREAD

**Discussion by paulearley** [esphome-econet/esphome-econet#273] [Ideas]
This integration was super fun and simple. I wound up creating a whole lovelace page to look at the data. I thought I would share it here. Thanks for a great integration!

![HomeAssistantPanel](https://github.com/esphome-econet/esphome-econet/assets/4460497/63e07313-1594-4116-adfb-e6a0b237a1e8)

**Comment by Ylianst** [comment]
Sharing mine. I have ESPHome-EcoNet along with the Moen Flow and a hot water re-circulation pump Zigbee switch. This integration is saving me a lot of money as I dynamically change my target temperature based on electricity rates.

![image](https://github.com/esphome-econet/esphome-econet/assets/1319013/9e83e9d0-5b6e-43c4-96f4-b4d04251791e)

![image](https://github.com/esphome-econet/esphome-econet/assets/1319013/1475b96d-f57a-4003-9eef-3c9deb1d69cb)
